Source/FixedBackground/KerrSchildFixedBG.hpp

```diff
@@ -0,0 +1,275 @@
+/* GRChombo
+ * Copyright 2012 The GRChombo collaboration.
+ * Please refer to LICENSE in GRChombo's root directory.
+ */
+
+#ifndef KERRSCHILDFIXEDBG_HPP_
+#define KERRSCHILDFIXEDBG_HPP_
+
+#include "ADMFixedBGVars.hpp"
+#include "Cell.hpp"
+#include "Coordinates.hpp"
+#include "DimensionDefinitions.hpp"
+#include "Tensor.hpp"
+#include "TensorAlgebra.hpp"
+#include "UserVariables.hpp" //This files needs NUM_VARS - total number of components
+#include "simd.hpp"
+
+//! Class which computes the initial conditions for a Kerr Schild BH
+//! https://arxiv.org/pdf/gr-qc/9805023.pdf
```

Comment:
Maybe also reference our paper with Jamie since this reference had some typos that we fixed.